python/make_variable.h

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Is this file just moved code are were there any changes?

Reply:
just moved code so i could also use it in `dataset.cpp`